data/tools/nginx-amplify.json

```diff
@@ -0,0 +1,21 @@
+[
+  {
+    "slug": "monitoring",
+    "name": "NGINX Amplify",
+    "description": "Amplify is free monitoring tool for NGINX. Amplify offers extended NGINX metric collection, visualization, monitoring, and a static analyzer for NGINX configuration. It's simple to set up, yet powerful enough to bring the much needed insight into NGINX performance. Amplify is a SaaS developed and operated by Nginx, Inc. Keep NGINX up to date and secure with a native monitoring solution.",
+    "url": "https://amplify.nginx.com",
+    "tags": [
+      "linux",
+      "bsd",
+      "free",
+      "commercial",
+      "ci", "cd",
+      "virt",
+      "cloud", "cloud-paas", "paas",
```

Comment:
And I believe cloud/paas tags not applicable as its not a paas platform (we use that for platforms like heroku/etc)